### ISSUE TITLE
Show only grey Image.

### ISSUE THREAD

**Issue by Dj-Redix** [nextcloud/unsplash#109]
use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

- [x] This is not a feature request
- [x] This is not an image-provider request
- [x] I have searched the issuetracker if the issue is not already reported.

**Describe the bug:**

On Dashboard i only see a **grey Image**. 

If I also check the box (_use a random image as the background of the navigation bar_) then it works to some extent. **It's not stable**. Sometimes a random image appears on the dashboard, other times it doesn't. **Image in navigation bar is always displayed**. 

If you uncheck (_use a random image as the background of the navigation bar_) and leave the other one at (_use a random image as the background of the dashboard_), I have a ****gray image** on the dashboard again and again**.

I generally **only checked** the **random image on the dashboard**.

I've already tried clearing the browser cache.
Restarting the server didn't work either.
Disabling, uninstalling and then reinstalling the Splash app didn't solve the problem either.

The problem is **since v2.0.0** and **also in v2.0.1**
With **v1.2.5** there were no problems.

**To Reproduce:**
1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

**Expected behavior:**

**Screenshots:**

**Server:**
 - Nextcloud version: NextCloudPi 24.0.6
 - Splash-App version: 2.0.1

**Additional Information:**

**Comment by newhinton** [comment]
Hm i have not recieved any other complaints about this, and i have a hard time replicating this. Could you replicate this with a different browser? Sometimes caching can break things from working. Also, can you check that your browser tries to load the images? I assume your nextcloud-instance is locally hosted on a pi, but your browser needs to fetch the images from the web. If your network connection is slow, loading can take a while.

**Comment by Dj-Redix** [comment]
I've tried a few more tests. Other browsers (Safari, Google Chrome) did not work.

I run 2 identical NextcloudPi's (hardware and software) at two different locations. One has 3 users, the other has 4.

I found the following.
It only works for one user on each instance. This user is always the same. As already described above: If only the function below is ticked (Use a random image as background of the dashboard) it only works for one user of each instance.

If I tick the last two (Random Image for Dashboard and Navigation Bar) then it works for all users on every instance.

So at the moment I have no other choice but to use the navigation bar with random images as well, although I don't really want that, since there are sometimes images that are very bright and the buttons can sometimes not be seen.

Many thanks for the help. Maybe an update will bring a solution for my problem in the future.
